docs/clustering/logical-replication-setup.rst

```diff
@@ -0,0 +1,146 @@
+.. _logical_replication_setup:
+
+==================================================
+Logical replication setup between CrateDB clusters
+==================================================
+
+Logical replication is a method of data replication between multiple clusters.
+As a publish/subscribe model, it allows a publishing cluster to make certain
+tables available for subscription. Subscribing clusters subscribe to a
+publication to replay any changes to the data on their side.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+.. _requirements:
+
+Requirements
+============
+.. NOTE::
+
+  Logical replication is available in CrateDB 4.8 and later.
+
+To replicate data from one cluster to another, we first of all need two CrateDB
+clusters. For this example, we spin them up locally as two single-node Docker
+containers:
+
+.. code-block:: console
+
+  sh$ docker run \
+        --name cluster-1 \
+        --detach \
+        --publish 4201:4200 \
+        --publish 5433:5432 \
+        crate/crate:nightly-4.8.0-20220401 \
+        -Cdiscovery.type=single-node
+  sh$ docker run \
+        --name cluster-2 \
+        --detach \
+        --publish 4202:4200 \
+        --publish 5434:5432 \
+        crate/crate:nightly-4.8.0-20220401 \
+        -Cdiscovery.type=single-node
+
+We apply an offset to the ports of each cluster, so they don't overlap. The
+Admin UIs of ``cluster-1`` and ``cluster-2`` are accessible at
+http://localhost:4201 and http://localhost:4202 respectively. All SQL
+statements discussed below can be executed via the corresponding Admin UI.
+
+There exists no fixed designation of a cluster being a subscriber or publisher.
+One cluster can even be both for different tables. Nevertheless, we will stick
+for this tutorial to ``cluster-1`` taking the role of the publishing cluster
+(data source). ``cluster-2`` will subscribe to ``cluster-1`` (data sink).
```

Comment:
I think a more telling name for the two clusters would be helpful further down in the article to understand their respecitve role.
Suggestions: `cluster-leader` & `cluster-follower` / `cluster-publisher` & `cluster-subscriber` / something similar

Reply:
Thought about that as well, but it slightly contradicts a statement further up that a cluster doesn't have a designated subscriber/publisher role (on the cluster level), but can actually be both, depending on the table.
I changed the names and rephrased the paragraph above, to indicate that this is a simplification.